Update Miner Binaries

Minerx117 opened the pull request:
30/01/2020

Minerx117 committed:
Update and rename CcminerZenemy-v2.3.ps1 to CcminerZenemy-v2.4.ps1

Minerx117 committed:
Update and rename BMiner-v15.8.7.ps1 to BMiner-v16.0.2.ps1

Minerx117 committed:
Update BMiner-v16.0.2.ps1

bminer is win10 only with current version

Minerx117 committed:
Update and rename Gminer-v1.92.ps1 to Gminer-v1.96.ps1

Minerx117 committed:
Update and rename Jayddee-v3.11.0.ps1 to Jayddee-v3.11.7.ps1

Minerx117 committed:
Update NanoMiner-v1.6.2.ps1

Minerx117 committed:
Rename NanoMiner-v1.6.2.ps1 to NanoMiner-v1.7.3.ps1

Minerx117 committed:
Update and rename PhoenixminerEthash-v4.8c.ps1 to PhoenixminerEthash-v4.9c.ps1

Minerx117 committed:
Update and rename SRBMinerMulti-v0.2.0.ps1 to SRBMinerMulti-v0.3.1.ps1

Minerx117 committed:
Update and rename XmrigCryptonight-v2.14.4-CPU.ps1 to XmrigCryptonight-v5.5.1-CPU.ps1

Minerx117 committed:
Update and rename XmrigCryptonight-v2.15.4-NVIDIA.ps1 to XmrigCryptonight-v5.5.1-NVIDIA.ps1

Minerx117 committed:
Update and rename XmrigCryptonight-v2.14.6-AMD.ps1 to XmrigCryptonight-v5.5.1-AMD.ps1

Minerx117 committed:
Update lolMiner-v0.9.3.ps1

Minerx117 committed:
Rename lolMiner-v0.9.3.ps1 to lolMiner-v0.9.7.ps1

Minerx117 committed:
Update and rename TTMiner-v3.0.10.ps1 to TTMiner-v3.1.1.ps1

Minerx117 committed:
Create Ccminersha3d-v1.0.ps1

Minerx117 committed:
Create Ccmineryescryptr-v5.ps1

Minerx117 committed:
Create Ccminerlyra2z330-v3.ps1

Minerx117 committed:
Update Ccmineryescryptr-v5.ps1

Minerx117 committed:
Update lolMiner-v0.9.7.ps1

Cuckatoo32 (NiceHash)

Minerx117 committed:
Update lolMiner-v0.9.7.ps1

AMD atm

Minerx117 committed:
Update BMiner-v16.0.2.ps1

win10 needs 10+gb

Minerx117 committed:
Update XmrigCryptonight-v5.5.1-NVIDIA.ps1

Minerx117 committed:
Update XmrigCryptonight-v5.5.1-AMD.ps1

aaronsace reviewed (commented):
Awesome, I'll check it later and update the release.

Minerx117 committed:
Update and rename MiniZEquihash-v1.5s.ps1 to MiniZEquihash-v1.5t.ps1

Major improvements: 150,5 and 210,9. Up to 4% depending on algo and GPU.
Also for RTX 2070 (up to 8% on 192,7), 1660 Ti.
Minor improvements in all other algos. Up to 2% depending on algo and GPU.
Better support for GTX 1660 Ti and GTX 1070.
Improved stability.

Minerx117 committed:
Update and rename XmrigCryptonight-v5.5.1-CPU.ps1 to XmrigCryptonight-v5.5.3-CPU.ps1

Minerx117 committed:
Update and rename XmrigCryptonight-v5.5.1-NVIDIA.ps1 to XmrigCryptonight-v5.5.3-NVIDIA.ps1

Minerx117 committed:
Update and rename XmrigCryptonight-v5.5.1-AMD.ps1 to XmrigCryptonight-v5.5.3-AMD.ps1

aaronsace commented on MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1:
```diff
@@ -27,18 +27,10 @@ if ($CUDAVersion -and [System.Version]$CUDAVersion -lt [System.Version]$Required
-    $HashSHA256 = "9450E5483CBBDBEBDB706087E5B468FB716028751667C3F663E07395DCBC0476"
-    $Uri = "https://github.com/xmrig/xmrig-nvidia/releases/download/v2.14.5/xmrig-nvidia-2.14.5-cuda10_1-win64.zip"
+ 
+if { 
```
Mistake?

Minerx117 replied:
not really your so far behide with xmrig it needs to be re written..

Minerx117 replied:
update your algos ie randomx randomsfx zergpool  im not going to do it for you

Minerx117 replied:
i guess so

Minerx117 replied:
cant be fucked to be honest

Minerx117 replied:
algos need updating

Minerx117 replied:
fixed the cuda warning Mistake. 
new xmrig is cuda 10.1 only xmrig.ps1 needs to be re written to fix --algo selection and api is not longer working. this is beyond my capabilities

aaronsace replied:
Maybe leave this one out for now...? 

Can always tackle this binary on a separate pull request...

Are all the other files complete?

Minerx117 replied:
done, all complete

Minerx117 replied:
running through test rigs atm all good so far,
need to add Verushash algo support ZergPool CPU (most profitable atm)
i'll do it in a couple hours

Minerx117 committed:
Update XmrigCryptonight-v5.5.3-NVIDIA.ps1

fix cuda warning

Minerx117 committed:
Update XmrigCryptonight-v5.5.3-NVIDIA.ps1

Minerx117 committed:
Update XmrigCryptonight-v5.5.3-NVIDIA.ps1

Minerx117 committed:
revert xmrig to old version 

api (port) and -a (--algo) require fixing for new versions of xmrig to work,
recommend re write of xmrig.ps1 file and change config file to suit.

Minerx117 committed:
revert xmrig to old version

api (port) and -a (--algo) require fixing for new versions of xmrig to work,
recommend re write of xmrig.ps1 file and change config file to suit.

Minerx117 committed:
revert xmrig to old version

api (port) and -a (--algo) require fixing for new versions of xmrig to work,
recommend re write of xmrig.ps1 file and change config file to suit.

Minerx117 committed:
Update Gminer-v1.96.ps1

Enable cuckaroom algo NiceHash

Minerx117 committed:
Update BMiner-v16.0.2.ps1

prefer Gminer 
may want to check API support for bminer and cuckaroom (was not working on my test rigs Api could not retrieve hashrate) after 3 attempts

Minerx117 committed:
JCE disable fix random algo variants

JCE disable fix random algo variants
CPU (SRBMiner randomx,randomarq,randomsfx) ZergPool

Minerx117 committed:
Update Algorithms.txt

cryptonightgpu

Minerx117 committed:
Update Ccmineryescryptr-v5.ps1

fix yescryptr algos

Minerx117 committed:
Update Ccminerlyra2z330-v3.ps1

Minerx117 committed:
Update SRBMinerMulti-v0.3.1.ps1

longer benchmark for random variants
